src/utils/getNextAvailableCode.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Returns 1 higher than the highest code,
+ * e.g. R1, R2, R5 => R6
+ */
+export const getNextAvailableCode = (itemList, prefix) => {
```

Comment:
This looks good. Not really related to this PR, but I'm not sure why the old app is of using an incremental code in the configurations. It seems like it would be simpler (and less prone to bugs) to just generate a random code?

Reply:
Agreed -- ticket created: https://dhis2.atlassian.net/browse/RWDQA-66